packages/TorneloScoresheet/src/components/Toolbar/Toolbar.tsx

```diff
@@ -30,17 +35,30 @@ const colourForMode: Record<AppMode, ColourType> = {
   [AppMode.TablePairing]: colours.primary,
 };
 
+const arbiterModeDisplay: Record<AppMode, 'none' | undefined> = {
+  [AppMode.EnterPgn]: 'none',
+  [AppMode.PariringSelection]: 'none',
+  [AppMode.TablePairing]: undefined,
+  [AppMode.GraphicalRecording]: undefined,
+  [AppMode.ResultDisplay]: undefined,
+};
+
```

Comment:
Any reason for mapping to 'none' | undefined? why not map to a boolean true/false for each mode?